documentation/performance/sizing-search.html

```diff
@@ -1,705 +1,581 @@
+Given a fixed occupancy (users, load clients) this 
+relationship between query throughput and latency is described by 
+<a href="https://en.wikipedia.org/wiki/Little%27s_law">Little's law</a> 
+, more on this in <a href="#content-cluster-scalability-model">content cluster 
+scalability model</a> section of this document.</p>
+
+<figure>
+<figcaption><b>Grouped content distribution</b></figcaption>
+<img src="../img/grouped_content_cluster.svg" height="420" width="840"/>
+</figure>
+
+<p>With a grouped distribution, content is distributed to a configured set of <em>groups</em>, 
+such that the entire document collection is contained in each group.
+A <em>group</em> contains a set of content nodes which the content is 
+distributed over using the Vespa <a href="../content/idealstate.html">distribution algorithm</a>. In the above illustration
+there are 4 nodes in total, 2 groups with 2 nodes in each group. <em>redundancy</em> is 2 and <em>searchable-copies</em> is also
+2. As can seen from the figure with this grouped configuration the content nodes only have a populated ready sub-database.
+A query is dispatched in parallel to all nodes in <b>one group</b> at a time 
+using a <a href="../reference/services-content.html#dispatch-policy">dispatch-policy</a>, 
+default is <em>round-robin</em> which gives an uniform query
+distribution over the set of groups.</p>
+
+<h3 id="high-availability">High Data Availability</h3>
+<p>Ideally we want our data to be available and searchable at all times even if there are node failures,
+ but high availability also costs resources as we need to replicate data. How many replicas of the data 
+we configure depends on what kind of availability guarantees the deployment should provide.  
+Vespa allows configuring the availability versus cost using the 
+<em>redundancy</em> and <em>searchable-copies</em> settings, described below: </p>
+
+<table class="table">
+  <thead></thead><tbody>
+    <tr id="redundancy">
+      <th>Redundancy</th>
+      <td>
+        <p>
+        <a href="../reference/services-content.html#redundancy">Redundancy</a>
+        Defines the total number of copies of each piece of data the cluster will store and maintain to avoid data loss. 
+        Example: with a redundancy of 2, the system tolerates 1 node failure before data becomes unavailable. 
+        </p>
+      </td>
+    </tr><tr id="searchable-copies">
+      <th>Searchable copies</th>
+      <td>
+        <p>
+        <a href="../reference/services-content.html#searchable-copies">Searchable-copies</a>
+        configures how many of the copies (as configured with <em>redundancy</em>) to be indexed 
+        (<em>ready</em>) at any time. Configuring
+        <em>searchable-copies</em> to be less than redundancy saves resources (memory, disk, cpu) 
+        as not all copies are indexed (<em>ready</em>).  In case 
+        of node failure the remaining nodes needs to activate (index) the 
```

Comment:
Technically, activating documents is a separate operation from indexing them (we activate ready replicas to un-blacklist them), so may consider just rewriting to "(...) the remaining nodes need to index the (...)"